boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -138,23 +155,99 @@ TEST_F(
     FsmStateMachineTest,
     check_if_primary_and_secondary_flow_rates_are_broadcasted_over_can_in_all_states)
 {
-    float fake_frequency = 1.0f;
+    float fake_flow_rate = 1.0f;
```

Comment:
Unrelated to wheel speed, but I corrected the label for this float since we are acquiring the flow rate, not a frequency